dump/config.json

```diff
@@ -0,0 +1,870 @@
+{
```

Comment:
These mappings look like they should be in the mappings wiki rather than the source repo. Or maybe that's too much work for now and we can do that later. What do you think?